### PR TITLE
add our names to the template license

### PR DESCRIPTION
**Pull request by konnov**
Since our LICENSE file was simply a template, I've added our names, as required.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -175,18 +175,7 @@
...
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2024] Igor Konnov, Jure Kukovec, Andrey Kuprianov, Thomas Pani.
```
Comment (thpani) [inline LICENSE:178]: I think the brackets are meant to be deleted:

```suggestion
   Copyright 2024 Igor Konnov, Jure Kukovec, Andrey Kuprianov, Thomas Pani.
```

**Review by thpani — APPROVED** [review]

**Comment by konnov** [comment]
Fixed that, @thpani please approve once again :)

**Comment by konnov** [comment]
I have changed the PR rules so they do not reset an approval on a new commit

**Comment by konnov** [comment]
Actually, I will close this PR, as it is superseded by #17

**Pull request closed without merging**